tests/test_forum.py

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/env python
+#
+# A library that provides a Python interface to the Telegram Bot API
+# Copyright (C) 2015-2022
+# Leandro Toledo de Souza <devs@python-telegram-bot.org>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser Public License for more details.
+#
+# You should have received a copy of the GNU Lesser Public License
+# along with this program.  If not, see [http://www.gnu.org/licenses/].
+import pytest
+
+from telegram import ForumTopic, ForumTopicClosed, ForumTopicCreated, ForumTopicReopened, Sticker
+
+TEST_TOPIC_ICON_COLOR = 0x6FB9F0
+TEST_TOPIC_NAME = "Sad bot true: real stories"
+# TODO replace with something meaningful when getForumTopicIconStickers is implemented?
+TEST_TOPIC_EMOJI_ID = "some_id"
+
+
+@pytest.fixture
+async def create_and_delete_topic(bot, forum_group_id):
+    # TODO rework with methods and attributes instead of dict keys
+    result = await bot._post(
+        "createForumTopic",
+        {"chat_id": forum_group_id, "name": TEST_TOPIC_NAME},
+    )
+
+    yield result
+
+    result = await bot._post(
+        "deleteForumTopic",
+        {"chat_id": forum_group_id, "message_thread_id": result["message_thread_id"]},
+    )
```

Comment:
now that the method is implemented upstream, let's use it here

Reply:
is it? I'm struggling to find it in the branch `api_6.3_update`